README.md

```diff
@@ -20,10 +20,18 @@ To build or test the plugin using IntelliJ:
 
 * Get set up with the pre-requisites listed here: http://www.jetbrains.org/intellij/sdk/docs/tutorials/custom_language_support/prerequisites.html
 * Create a new IntelliJ project using this repo as the existing source.
+* Make sure the repo is using the IntelliJ plugin java SDK 
+* Make sure the repo has a valid Scala SDK attached as a project dependency.
+  * Otherwise you'll see errors like `"Cannot find class WdlElementType"` even though it's clearly there!
 * Generate the necessary files (on Mac):
- * Navigate to wdl.bnf and generate sources using \[Command + G\]
- * Navigate to Wdl.flex and generate sources using \[Command + G\]
-* At this point, you can run or test the project using IntelliJ's preset run modes. Fuller instructions for setting this up are in the IntelliJ links above.
+  * Navigate to Wdl.flex and generate sources using \[Command + Shift + G\] 
+  * Navigate to wdl.bnf and generate sources using \[Command + Shift + G\]
+* At this point, you can run or test the project using IntelliJ's preset run modes. 
+  * Open the `Run Configurations` window (for me, in the top right of the IntelliJ window) 
+  * Add a new one configuration with the `+` icon.
+  * Choose the '`Plugin`' type. The defaults should be good.
+  * When this is run, a new IntelliJ window opens with the in-development plugin preinstalled.
+  * Fuller instructions for setting this up are in the IntelliJ links above.
```

Comment:
🐦  I'm not sure "fuller" is a word (in this context). 
I'd recommend "Additional instructions for setting this up are in the IntelliJ links above."

ps I looked it up and "fuller" is a grooved or rounded tool on which iron is shaped.